packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -253,6 +272,47 @@ export default class Meeting extends StatelessSparkPlugin {
     this.correlationId = id;
   }
 
+  startAudioVideo(mediaDirection) {
+    this.audio.init({mute: !mediaDirection.sendAudio, self: true}, this);
+    this.video.init({mute: !mediaDirection.sendVideo, self: true}, this);
+  }
+
+  muteAudio() {
+    const options = {};
+    options.mute = true;
+    options.self = true;
```

Comment:
Is there a reason you broke these out instead of writing

```
this.toggleAudio({
  mute: true,
  self: true
})
```

Reply:
nope -- changed

Reply:
done